<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyUuidWidget</class>
 <widget class="QWidget" name="MyUuidWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>540</width>
    <height>423</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="m_uuidLabel">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>UUID: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="m_lineEdit">
       <property name="font">
        <font>
         <family>.Lucida Grande UI</family>
        </font>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QToolButton" name="m_removeToolButton">
         <property name="text">
          <string>-</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="m_addToolButton">
         <property name="text">
          <string>+</string>
         </property>
         <property name="iconSize">
          <size>
           <width>16</width>
           <height>16</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="m_generatePushButton">
       <property name="text">
        <string>Generate</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListWidget" name="m_listWidget"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
